crates/primitives/src/transaction/tx_value.rs

```diff
@@ -19,7 +19,7 @@ use proptest::{
 /// this full precision, hence we use a wrapper type to allow for overriding of encoding.
 #[add_arbitrary_tests(compact, rlp)]
 #[derive(Default, Debug, Copy, Clone, Hash, PartialEq, Eq, Serialize, Deserialize)]
-pub struct TxValue(U256);
+pub struct TxValue(pub U256);
```

Comment:
why do we need this now?
since this can be converted into U256, I don't think we need to make this pub

Reply:
addressed in https://github.com/anton-rs/op-reth/pull/134